README.md

```diff
@@ -0,0 +1,137 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/">HTML</a></li>
+    <li><a href="https://www.w3.org/TR/CSS/#css">CSS</a></li>
+    <li><a href="#">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Set up the linter for the project 👌
+- Add the node modules to the .gitignore file🚀
+- Add the styling
+- Use DOM Manipulation
+- Add modules using ES6 Syntaxes
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+
+1. Computer
+2. Internet
+3. Web Browser
+4. Git
+5. Code Editor
+
+### Setup
+
+Clone this repository to your desired folder Enter the following commands in the terminal:
+
+```sh
+  git clone git@github.com:efmomoh/my-books.git
+  cd awesome-books
+```
+
+### Install
+
+Install this project with the following command in the terminal:
+
+```sh
+ npm install 
+
+```
+
+### Usage
+
+To run the project,
+
+```sh
+  Open the browser and enter the address of the index.html file from your computer/server
+```
+
+### Run tests
+
+```sh
+   1. npx hint.
+   2. npx stylelint "**/*.{css,scss}"
+```
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo Link](https://efmomoh.github.io/my-books/)
+
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Enssah**
+
+- GitHub: [@githubhandle](https://github.com/efmomoh)
+- Twitter: [@twitterhandle](https://twitter.com/@efmomoh)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/efmomoh?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BQI%2F5GWZxS063VqRg2rilyg%3D%3D)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+- Add books
+- Remove books
+- Adds Author
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 contributing <a name="contributing"></a>
+
+You are welcome, and click <a href="https://github.com/efmomoh/my-books/issues">here</a> for contribution, issues, and feature request 🙏
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star and follow me on GitHub LinkedIn and Twitter
+, you are also welcome to any suggestions🙏
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Creative Commons for the hard work put in as a team for this awesome UX Design. Hence, this project was built based on Creative Commons legal designs. Thanks, team for the success!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] @efmomoh Your license link is not working, this is because you do not have a license file in this branch , kindly consider adding a license file and ensuring the link is working